apps/web/src/pages/invites/MembersInvitePage.tsx

```diff
@@ -88,6 +107,9 @@ export function MembersInvitePage() {
                   ) : (
                     <Tag>Member</Tag>
                   )}
+                  <span onClick={() => removeMemberClick(member)}>
+                    <Tag css={{ cursor: 'pointer' }}>X</Tag>
+                  </span>
```

Comment:
Could you please checkout the design at figma: 
https://www.figma.com/file/IpAw7sGWYZ8DHTem0tSM8r/Ready-for-Dev?node-id=28%3A7377

So we can actually have the 3 dots with some space around, and upon clicking we can use mantine's dropdown with a delete button. You can also see Workflow editor for example.

Wdyt?

Reply:
<img width="329" alt="CleanShot 2022-07-25 at 20 14 42@2x" src="https://user-images.githubusercontent.com/8872447/180836202-1e91b3ee-817c-45c1-b6fc-5a0093e4bf79.png">
Here is how it looks from the workflow editor.

Reply:
Thanks for the example, fixed it.
![image](https://user-images.githubusercontent.com/48491140/180950188-06bc0f6e-3624-46f3-92f0-e410c9ded6e4.png)